articles/110_interface_definition.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: default
+title: Interface definition
+permalink: articles/interface_definition.html
+abstract:
+  This article specifies the format to specify the data structures ROS 2 components exchange to interact with each other.
```

Comment:
Good point. The double "specified" was really confusing. I have updated the sentence of the abstract as well as in the first paragraph in the following commit: 5bd10d3fe6199c994934e8029f5932968d83edbc